pkg/virt-config/featuregate/active.go

```diff
@@ -0,0 +1,123 @@
+	DockerSELinuxMCSWorkaround = "DockerSELinuxMCSWorkaround"
+	VSOCKGate                  = "VSOCK"
+	// DisableCustomSELinuxPolicy disables the installation of the custom SELinux policy for virt-launcher
+	DisableCustomSELinuxPolicy = "DisableCustomSELinuxPolicy"
+	// KubevirtSeccompProfile indicate that Kubevirt will install its custom profile and
+	// user can tell Kubevirt to use it
+	KubevirtSeccompProfile = "KubevirtSeccompProfile"
+	// DisableMediatedDevicesHandling disables the handling of mediated
+	// devices, its creation and deletion
+	DisableMediatedDevicesHandling = "DisableMDEVConfiguration"
+	// HotplugNetworkIfacesGate enables the virtio network interface hotplug feature
+	HotplugNetworkIfacesGate = "HotplugNICs"
+	// PersistentReservation enables the use of the SCSI persistent reservation with the pr-helper daemon
+	PersistentReservation = "PersistentReservation"
+	// VMPersistentState enables persisting backend state files of VMs, such as the contents of the vTPM
+	VMPersistentState = "VMPersistentState"
+	MultiArchitecture = "MultiArchitecture"
+	// VMLiveUpdateFeaturesGate allows updating certain VM fields, such as CPU sockets to enable hot-plug functionality.
+	VMLiveUpdateFeaturesGate = "VMLiveUpdateFeatures"
+	// When BochsDisplayForEFIGuests is enabled, EFI guests will be started with Bochs display instead of VGA
+	BochsDisplayForEFIGuests = "BochsDisplayForEFIGuests"
+	// NetworkBindingPlugingsGate enables using a plugin to bind the pod and the VM network
+	NetworkBindingPlugingsGate = "NetworkBindingPlugins"
+	// AutoResourceLimitsGate enables automatic setting of vmi limits if there is a ResourceQuota with limits associated with the vmi namespace.
+	AutoResourceLimitsGate = "AutoResourceLimitsGate"
+
+	// Owner: @lyarwood
+	// Alpha: v1.1.0
+	//
+	// CommonInstancetypesDeploymentGate enables the deployment of common-instancetypes by virt-operator
+	CommonInstancetypesDeploymentGate = "CommonInstancetypesDeploymentGate"
+	// AlignCPUsGate allows emulator thread to assign two extra CPUs if needed to complete even parity.
+	AlignCPUsGate = "AlignCPUs"
+
+	// VolumesUpdateStrategy enables to specify the strategy on the volume updates.
+	VolumesUpdateStrategy = "VolumesUpdateStrategy"
+	// VolumeMigration enables to migrate the storage. It depends on the VolumesUpdateStrategy feature.
+	VolumeMigration = "VolumeMigration"
+	// Owner: @xpivarc
+	// Alpha: v1.3.0
+	//
+	// NodeRestriction enables Kubelet's like NodeRestriction but for Kubevirt's virt-handler.
+	// This feature requires following Kubernetes feature gate "ServiceAccountTokenPodNodeInfo". The feature gate is available
+	// in Kubernetes 1.30 as Beta.
+	NodeRestrictionGate = "NodeRestriction"
+)
+
+func init() {
+	RegisterFeatureGate(FeatureGate{Name: ExpandDisksGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: CPUManager, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: NUMAFeatureGate, State: Alpha})
```

Comment:
This has GA'd with your recent rebase and can be dropped, you can move the definition to inactive.go as well.